### PR TITLE
Events UI

### PR DESCRIPTION
**Pull request by khavinshankar**
- [x] add UI
- [x] wire api with backend

### CODE REVIEW

File: src/Components/Facility/ConsultationDetails/Events/EventsList.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+interface Props {
+  consultation: ConsultationModel;
+}
+
+export default function EventsList({ consultation }: Props) {
+  const [consultationId] = useSlugs("consultation");
+  const { t } = useTranslation();
+
+  console.log("consultation", consultation, consultationId);
```
Comment (Ashesh3) [inline src/Components/Facility/ConsultationDetails/Events/EventsList.tsx]:
```suggestion
```

File: src/Components/Facility/ConsultationDetails/Events/EventsList.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                  >
+                    {(() => {
+                      switch (item.event_type.name) {
+                        case "INTERNAL_TRANSFER":
+                        case "CLINICAL":
+                        case "DIAGNOSIS":
+                        case "ENCOUNTER_SUMMARY":
+                        case "HEALTH":
+                        default:
+                          return <GenericEvent event={item} />;
```
Comment (Ashesh3) [inline src/Components/Facility/ConsultationDetails/Events/EventsList.tsx:63]: Are these just here as placeholders? Just checking if this was intentional to let all these cases fall to the default case.

**Review by Ashesh3 — APPROVED** [review]

File: src/Components/Facility/ConsultationDetails/Events/EventsList.tsx (hunk 2 above)
Comment (khavinshankar) [reply src/Components/Facility/ConsultationDetails/Events/EventsList.tsx:63]: It was intensional, the plan was to create a separate card for each type of event down the line

File: src/Components/Facility/ConsultationDetails/ConsultationUpdatesTab.tsx [hunk 3]
```diff
@@ -665,7 +668,26 @@ export const ConsultationUpdatesTab = (props: ConsultationTabProps) => {
...
         </div>
         <div className="w-full pl-0 md:pl-4 xl:w-1/3">
-          <DailyRoundsList consultation={props.consultationData} />
+          <SwitchTabs
+            className="mt-3 w-full lg:w-full"
+            tab2={
+              <div className="flex items-center justify-center gap-1 text-sm">
+                Events
+                <span className="rounded-lg bg-warning-400 p-[1px] px-1 text-[10px] text-white">
+                  in beta
```
Comment (rithviknishad) [inline src/Components/Facility/ConsultationDetails/ConsultationUpdatesTab.tsx]:
```suggestion
                  beta
```

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
Fields currently we support : https://github.com/coronasafe/care/blob/master/care/facility/management/commands/load_event_types.py

**Comment by nihal467** [comment]
- [ ] daily round symptoms are shown are numbers
- [x] UI is breaking for long doctor notes
- [ ] consultation and log update modification are not been shown except IP number

**Comment by nihal467** [comment]
LGTM

**Pull request merged**